uktrade/lite-frontend
# Added govuk-country-and-territory-autocomplete to match functionality

Searching a country was returning as thought it was doing a "in" function in python rather than a "startswith".

[this](https://github.com/alphagov/govuk-country-and-territory-autocomplete) was used to replicate the "startswith" functionality and allow for regional/territorial querying.

To test:
go to the exporter,
make a new application,
add end user information,
URL should look like this:
* http://localhost:8300/applications/<APPLICATION_ID>/end-user/set/

[LTD-3206](https://uktrade.atlassian.net/browse/LTD-3206)


[LTD-3206]: https://uktrade.atlassian.net/browse/LTD-3206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
